addons/sale/tests/test_sale_to_invoice.py

```diff
@@ -198,6 +198,45 @@ def test_downpayment_line_remains_on_SO(self):
         sale_order.invoice_ids.action_post()
         self.assertEqual(downpayment_line[0].price_unit, 50, 'The down payment unit price should not change on SO')
 
+    def test_downpayment_line_name(self):
```

Comment:
This test doesn't cover any of your changes anymore, it could be removed from your PR and commit.
Ideally, for your fix, the test should be in `pos_sale` and mimic the flow you're fixing as much as possible.